_source/_posts/2023-07-17-enterprise-ready-getting-started.md

```diff
@@ -0,0 +1,148 @@
+2. **Sync all your customer's users with SCIM** - Give your app the complete picture of your customer's user base using [System for Cross-domain Identity Management (SCIM)](https://scim.cloud/).
+3. **Add automated reporting with no-code Okta Workflows** - Enable your customers to automate actions within your app using a low/no-code configuration mechanism.
+4. **Automate Okta management with Terraform** - Automate environment creation and ensure consistency in managing your Okta organization using Terraform.
+
+You can pick which workshop to participate in and the order in which you participate. Choose the workshops that best address your next most pressing need.
+
+> We want to ensure you can follow the workshops and focus on understanding how to enhance the application for enterprise use cases, so don't use this base application as a template expecting a production-quality B2C or B2B application - it has some obvious intentional security shortcomings!
+
+{% include toc.md %}
+
+## Build an enterprise-ready SaaS application using ReactJS, Express APIs, and Prisma
+
+While SaaS applications run a range of tech stacks, we'll focus on a SPA frontend with an API backend for this workshop series.
+
+This application uses [React](https://react.dev/) for the frontend application and [Express](https://expressjs.com/) to build the API backend in [TypeScript](https://www.typescriptlang.org/). We'll need persistent storage, so you'll use [SQLite](https://www.sqlite.org/index.html) for a local database. To make inspecting and changing the database more effortless, you'll use an ORM that comes with a database visualization tool: [Prisma](https://www.prisma.io/). The entire project is in a [Nx workspace](https://nx.dev/) so you can manage the whole application with one set of dependency instructions and scripts to serve the app.
+
+Do you want to see your framework of choice in a workshop? Let us know your preferred tech stack for an enterprise-ready SaaS application in the comments below!
+
+## Install NodeJS and required tools
+
+Each tech stack has the required tooling to install on your machine. Depending on the specific workshop you're participating in, you may need some knowledge of JavaScript, REST concepts, and command line operations in addition to the installed tools.
+
+Next, you'll install the required tools, get a local copy of the project, and walk through the commands to start and stop the application.
+
+There's a [companion video for this post](https://youtu.be/UZQ_ke8BuRU) that shows the installation process and running the application. 
+
+{% youtube UZQ_ke8BuRU %}
+
+### Install NodeJS, npm, and npx
+
+You'll need [Node](https://nodejs.org/en) v18. When you install Node, you'll automatically get npm, a command line tool to install, update, and manage dependencies. npm includes a command called npx, allowing you to run a npm package without installing it locally.
```

Comment:
```suggestion
You'll need [Node](https://nodejs.org/en) v18. When you install Node, you'll automatically get npm, a command line tool to install, update, and manage dependencies. Npm includes a command called npx, allowing you to run a npm package without installing it locally.
```

Reply:
@coltonbeach and I discussed - we will leave lowercase as that's the casing npm uses for their internal documentation (even at the beginning of a sentence, as shown here https://docs.npmjs.com/policies/terms#paid-npm-services)